app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    input_command, file_in, file_out = command.split()
+    if command == "cp":
+        with (
+            open(file_in, "r") as file_to_copy,
+            open(file_out, "a+") as copy_of_file
+        ):
+            lines = file_to_copy.readlines()
+            for line in lines:
+                copy_of_file.write(line)
```

Comment:
what sense to copy, if we have both names equal?)

Reply:
> what sense to copy, if we have both names equal?)

please explain in more detail. Do I need to add another check for filename equality?

Reply:
if you have `file_in == file_out`, what sense to copy?

Reply:
Ok, will add an additional condition